src/main/java/net/william278/velocitab/packet/TeamsPacketAdapter.java

```diff
@@ -33,6 +34,7 @@
 @RequiredArgsConstructor
 public abstract class TeamsPacketAdapter {
 
+    private final Velocitab plugin;
```

Comment:
why?

Reply:
It's needed [here](https://github.com/WiIIiam278/Velocitab/pull/119/files#diff-7c65849dd50720a63c89094ad268331b20458b6d9b4fef371d589916d8d36255R101)